atom/browser/native_window_mac.mm

```diff
@@ -1205,6 +1211,67 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
   return collectionBehavior & NSWindowCollectionBehaviorCanJoinAllSpaces;
 }
 
+void NativeWindowMac::SetVibrancy(const std::string& type) {
+  if (!(base::mac::IsOSMavericks() || base::mac::IsOSYosemiteOrLater())) return;
+
+  NSView* vibrant_view = [window_ vibrantView];
+
+  if (type.empty()) {
+    if (vibrant_view == nil) return;
+
+    [vibrant_view removeFromSuperview];
+    [window_ setVibrantView:nil];
+
+    return;
+  }
+
+  NSVisualEffectView* effect_view = (NSVisualEffectView*)vibrant_view;
+  if (effect_view == nil) {
+    effect_view = [[NSVisualEffectView alloc] initWithFrame:
```

Comment:
AFAIK we don't need `autorelease`, because it is already a `retain`ed property, and ARC will destroy it, when we set it to `nil` .